testutil/gulp-mocha-runner.js

```diff
@@ -78,7 +78,26 @@ var plugin = function (options) {
             }
 
             const chrome = await launchChrome();
-            const protocol = await CDP({port: chrome.port});
+
+            let protocol = null;
```

Comment:
This is from: https://github.com/discourse/discourse/blob/master/test/run-qunit.js
Found it on this issue report:
https://github.com/GoogleChrome/chrome-launcher/issues/145

Not sure why we didn't run into it before.